### PR TITLE
Add Understand Your Setup Size Guide

### PR DESCRIPTION
**Pull request by tenmozes**
Signed-off-by: Artem Navoiev <tenmozes@gmail.com>

### CODE REVIEW

File: docs/guides/understand-your-setup-size.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+### Churn Rate
+
+The higher the churn rate, the more compute resources are needed for the efficient work of VictoriaMetrics. It is recommended to lower the churn rate as much as possible. The tolerable churn rate is less than 5% of the number of active time series.
+
+The high churn rate is commonly a result of using high-volatile labels, such as `client_id`, `url`, `checksum`, `timestamp`, etc. In Kubernetes, the pod's name is also a volatile label because it changes each time pod is redeployed. For example, a service exposes 1000 time series. If we deploy 100 replicas of the service, the total amount of active time series will be 1000*100 = 100000. If we redeploy the service, each replica's pod name will change, and the number of active time series will double because all the time series will update the pod's name label.
+
+To track the churn rate in VictoriaMetrics, use the following query:
+
+**`sum(rate(vm_new_timeseries_created_total))`**
+
```
Comment (hagen1778) [inline docs/guides/understand-your-setup-size.md:44]:
```suggestion
Use the following query to track the churn rate for the last 24h in Prometheus:

**sum(sum_over_time(scrape_series_added[24h]))**

For VictoriaMetrics query would be the following:

**sum(increase(vm_new_timeseries_created_total[24h]))**
```

File: docs/guides/understand-your-setup-size.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+### Terminology
+
+- [Active Time Series](https://docs.victoriametrics.com/FAQ.html#what-is-an-active-time-series) - the [time series](https://docs.victoriametrics.com/keyConcepts.html#time-series) that receive at least one sample for the latest hour;
+- Ingestion Rate - how many [data points](https://docs.victoriametrics.com/keyConcepts.html#raw-samples) you ingest into the database per second;
+- [Churn Rate](https://docs.victoriametrics.com/FAQ.html#what-is-high-churn-rate) - how frequently a new time series is registered. For example, in the Kubernetes ecosystem, the pod name is a part of time series labels. And when the pod is re-created, its name changes and affects all the exposed metrics, which results in high cardinality and churn rate problem;
+- Query Per second - the number of [read queries](https://docs.victoriametrics.com/keyConcepts.html#query-data) per second;
+- Retention period - for how long data is stored in the database.
+
```
Comment (hagen1778) [inline docs/guides/understand-your-setup-size.md:12]:
```suggestion

For those who are familiar with Prometheus, we recommend reading [Prometheus storage: technical terms for humans](https://valyala.medium.com/prometheus-storage-technical-terms-for-humans-4ab4de6c3d48).

```

File: docs/guides/understand-your-setup-size.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+To track the churn rate in VictoriaMetrics, use the following query:
+
+**`sum(rate(vm_new_timeseries_created_total))`**
+
+
+### Ingestion Rate
+
+Ingestion rate is how many time series are pulled (scraped) or pushed per second into the database. For example, if you scrape a service that exposes 1000 time series with an interval of 15s, the ingestion rate would be 1000/15 = 66 [samples](https://docs.victoriametrics.com/keyConcepts.html#raw-samples) per second. The more services you scrape or the lower is scrape interval the higher would be the ingestion rate.
+For Ingestion Rate calculation, you need to know how many time series you pull or push and how often you save them into VictoriaMetrics. To be more specific, the formula is the Number Of Active Time Series / Metrics Collection Interval.
```
Comment (hagen1778) [inline docs/guides/understand-your-setup-size.md:49]:
```suggestion
For Ingestion Rate calculation, you need to know how many time series you pull or push in VictoriaMetrics per second. To be more specific, the formula is the Number Of Active Time Series / Metrics Collection Interval.
```

File: docs/guides/understand-your-setup-size.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# Understand Your Setup Size
+
+The docs provide a simple and high-level overview of Ingestion Rate, Active Timeseries, and Query per Second. These terms are a part of capacity planning ([Single-Node](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#capacity-planning), [Cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#capacity-planning)) and [Managed VictoriaMetrics](https://docs.victoriametrics.com/managed_victoriametrics/) pricing.
+
+### Terminology
```
[the remark this answers is not in the document]
Comment (hagen1778) [reply docs/guides/understand-your-setup-size.md:5]: Please note, ## is then used by docs engine to build the hierarchy tree. Skipping levels or having too much levels can look weird.

**Review by hagen1778 — CHANGES_REQUESTED** [review]
Please, use some spell-checking for this text to fix the typos.

File: docs/guides/understand-your-setup-size.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (tenmozes) [reply docs/guides/understand-your-setup-size.md:5]: I've done that intentionally Terminology is a sub-item of the first paragraph. Calculation is the next important item

[the remark this answers is not in the document]
Comment (hagen1778) [reply docs/guides/understand-your-setup-size.md:5]: So you have level-1 item. And the next item is level-3. Why not level-2?

[the remark this answers is not in the document]
Comment (tenmozes) [reply docs/guides/understand-your-setup-size.md]: fixed, thanks

**Review by hagen1778 — APPROVED** [review]
thank you!

**Pull request merged**